__tests__/cairov24onward.test.ts

```diff
@@ -375,4 +375,36 @@ describe('Cairo v2.4 onwards', () => {
       await expect(nonZeroContract.call('send_nonZero_u256', [myU512])).rejects.toThrow();
     });
   });
+
+  describe('Cairo v282 u96', () => {
```

Comment:
just a nitpick for consistency, other describe-s have a different version format (with dots), like 'Cairo **v2.6.0** ...'

Reply:
Modified.

Reply:
Modified.